packages/node/src/login/oidc/oidcHandlers/ClientCredentialsOidcHandler.spec.ts

```diff
@@ -246,14 +254,52 @@ describe("handle", () => {
     );
   });
 
-  it("throws if the issuer does not return an ID token", async () => {
+  // Note that this is a temporary fix, and it will eventually be removed from the
+  // codebase once the client credential use case is properly covered by the authentication
+  // panel.
+  it("gets the WebID from the access token if the issuer does not return an ID token", async () => {
     const tokens = mockDpopTokens();
     tokens.id_token = undefined;
     setupOidcClientMock(tokens);
+    setupGetWebidMock("https://my.webid/");
     const clientCredentialsOidcHandler = new ClientCredentialsOidcHandler(
       mockDefaultTokenRefresher(),
       mockStorageUtility({})
     );
+
+    const sessionInfo = await clientCredentialsOidcHandler.handle({
+      ...standardOidcOptions,
+      client: {
+        clientId: "some client ID",
+        clientSecret: "some client secret",
+        clientType: "static",
+      },
+    });
+    // The session's WebID should have been picked up from the access token in
+    // the absence of an ID token.
+    expect((sessionInfo as SolidClientAuthnCore.ISessionInfo).webId).toBe(
```

Comment:
I think we'd actually need to change `ISessionInfo` to something like:

```
type ISessionInfo = {
  isLoggedIn: false;
  sessionId: string;
} | {
  isLoggedIn: true;
  sessionId: string;
  webId: string;
  clientAppId?: string;
  expirationDate?: number;
}```

(note: I'm leaving clientAppId and expirationDate as optional as they're currently being refactored)